### ISSUE TITLE
multi-line return errors for a computed property

### ISSUE THREAD

**Issue by acidjazz** [vuejs/eslint-plugin-vue#743]
**Tell us about your environment**
nuxtjs v2.3.4
* **ESLint version:**	
^5.1.0
* **eslint-plugin-vue version:**	
5.0.0
* **Node version:**	
v10.12.0
**Please show your full configuration:**

```js
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
  extends: [
    'plugin:vue/recommended'
  ],
  // required to lint *.vue files
  plugins: [
    'vue'
  ],
  // add your custom rules here
  rules: {
    'no-unused-vars': 0,
    'quotes': 0,
    'comma-dangle': 0,
    'padded-blocks': 0,
    'no-multi-spaces': 0,
    'no-multiple-empty-lines': ['error', { "max": 2, "maxEOF": 2, "maxBOF": 2, }],
    'key-spacing': 0,
    'no-console': process.env.APP_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.APP_ENV === 'production' ? 'error' : 'off'
  },
}
```

**What did you do?**	

```js
  computed: {
    range () {
      return
        ((this.end_minutes * 60) + this.end_seconds)
        -
        ((this.start_minutes * 60) + this.start_seconds)
    },
  },
```

**What did you expect to happen?**
No errors

**What actually happened?**
```
error  Expected to return a value in "range" computed property
```
If i put the return all on one line the error goes away

**Comment by michalsnik** [comment]
Hey @acidjazz The ESLint is right about it. If you put the content of return in the new line, the [automatic semicolon insertion](https://dev.to/promhize/what-you-need-to-know-about-javascripts-automatic-semi-colon-insertion-78a) will insert semicolon after `return` and thus your computed property won't return any value.
